.github/workflows/test-chart.yml

```diff
@@ -0,0 +1,89 @@
+name: Test Chart
+
+on:
+  push:
+  pull_request:
+
+env:
+  # The actual version will be injected to the chart by the task that builds the chart
+  DUMMY_VERSION: 1.2.3
+
+jobs:
+  test-e2e:
+    name: Run on Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: Install the latest version of kind
+        run: |
+          curl -Lo ./kind https://kind.sigs.k8s.io/dl/latest/kind-linux-amd64
+          chmod +x ./kind
+          sudo mv ./kind /usr/local/bin/kind
+
+      - name: Verify kind installation
+        run: kind version
+
+      - name: Create kind cluster
+        run: kind create cluster
+
+      - name: Prepare integration-service
+        run: |
+          go mod tidy
+          export IMG=integration-service:v$DUMMY_VERSION
+          yq -i '.controllerManager.container.image = env(IMG)' dist/chart/values.yaml
+          make img-build IMG=$IMG
+          kind load docker-image $IMG
+
+      - name: Verify Helm installation
+        run: helm version
+
+      - name: Lint Helm Chart
+        run: |
+          helm lint ./dist/chart
```

Comment:
We're [using kube-linter](https://github.com/konflux-ci/integration-service/blob/c40254a2fd810b2e2a00d1467a9a989f544d05eb/.github/workflows/pr.yaml#L156-L158) to scan the manifests from `/config` from the PR workflow. I think we also need to run `kube-linter` in this workflow since its using a different set of manifests.

Reply:
I wonder if kube-linter will be able to lint Helm templates, as they are not exactly manifests. I'll look into that.

Reply:
Running `helm template` and scanning the result should take care of that.

Reply:
Reading now, I see that it supports Helm natively. I ran it and it actually helped me find a bug in one of the templates. I'll look into running it in CI.

Reply:
I just added it (and confirmed it to be able to pass and fail).